src/libraries/System.Runtime/tests/System/Diagnostics/StackTraceHiddenAttributeTests.cs

```diff
@@ -0,0 +1,141 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using Xunit;
+
+namespace System.Tests
+{
+    public class StackTraceHiddenAttributeTests
```

Comment:
```suggestion
    public class StackTraceHiddenAttributeTests
```
```suggestion
    public static class StackTraceHiddenAttributeTests
```

Reply:
Is your preference for most/all test methods to be static?  We're very inconsistent on that, but I *think* (I'd need to actually measure, as I could be wrong) we tended towards instance methods.

Reply:
I do not have a preference. I looked at two other random test files and both of them were static. Feel free to ignore it.

Reply:
Ok.  We lack consistency and guidance here.  I'd be happy for us to put some in place.  But let's do it separately.